app/Models/StudentInfo.php

```diff
@@ -26,4 +27,7 @@ class StudentInfo extends Model
         'academic_interest'
     ];
 
+    public function users(){
+        return $this->belongsTo('App\User', 'id', 'user_id');
+    }
```

Comment:
consider moving App\Users into the models folder when you get the chance

Reply:
agreed